tester/check_replacer.js

```diff
@@ -45,25 +45,25 @@ function generateTest(name, got, ifExpression, needWant, continueOnFailure, valu
   let expr;
   if (needWant) {
     expr = `
-    test('${name}', () => {
+    test('${name}', ():i32 => {
       const got = ${got};
       const want = ${value[iValue]};
       if (${ifExpression}) {
         error('${got} = ' + got.toString() + ', ' + want.toString() + ' was expected.');
-        return ${continueOnFailure ? '0':'-1'};
+        return ${continueOnFailure ? 'TestResult.Failure':'TestResult.StopTestSet'};
       }
-      return 1;
+      return TestResult.Success;
     });\n`;
     iValue++; // one value was used in template
   } else {
     expr = `
-    test('${name}', () => {
+    test('${name}', ():i32 => {
       const got = ${got};
       if (${ifExpression}) {
         error('${got} was ' + got.toString() + '.');
-        return ${continueOnFailure ? '0':'-1'};
+        return ${continueOnFailure ? 'TestResult.Failure':'TestResult.StopTestSet'};
```

Comment:
what is the `${}`? are we sure that it will return an i32?

Reply:
I'm using a [template literal](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Template_literals).
It will return TestResult.Failure or TestResult.StopTestSet which is an i32.

You can test it by playing with the __tests__/samples.spec.ts